### ISSUE TITLE
Blank tag remains on page after turning off "Show Title"

### ISSUE THREAD

**Issue by tw77** [pressbooks/pressbooks#2582]
### Description

Turning off the "Show Title” option in the edit panel for a page (under Status & Visibility) doesn't remove the title header tag from the saved page. Instead, this blank tag appears in the page’s HTML:

```<header> <h1 class="entry-title"> </h1> </header>```

If a chapter is numbered, the number appears in this tag, but if not, this tag is empty. Reported as a bug here: https://pressbooks.zendesk.com/agent/tickets/8675. The client writes: "This is a minor accessibility issue, as screen readers may announce the blank heading and page navigation may attempt to locate the focus on a heading item that cannot be seen by the user."

### Steps to Reproduce

1. Add a chapter to a book.
2. If chapter numbers are set to display in the book, change Chapter Type to Numberless.
3. Uncheck the Show Title checkbox.
4. Preview the chapter (or save and view).

**Expected behavior:** ```<header> <h1 class="entry-title"> </h1> </header>``` doesn't appear in the HTML for the resulting page.

**Actual behavior:** This blank tag appears.
